hiss/shared/test_case.py

```diff
@@ -1,16 +1,56 @@
+from django.core.files.uploadedfile import SimpleUploadedFile
 from django.test import TestCase
+from django.utils import timezone
 
 from user.models import User
+from application.models import Wave
 
+TEST_RESUME_DIR = "test_resume_dir"
 
```

Comment:
```suggestion
@override_settings(MEDIA_ROOT=TEST_RESUME_DIR)
```
Otherwise the `TEST_RESUME_DIR` variable will go unused. `override_settings` comes from `django.test`.